ui/layout-pages.md

```diff
@@ -5,13 +5,30 @@ title: Layout Pages
 # Layout Pages
 
 Layout Pages are used in the main web user interface to display widgets in an organized manner.
-You can choose among several layouts, based on whether you want to control the layout completely, or let the container decide where to place the widgets.
+You can choose between two different basic types of layouts:
 
-![Layout Page Example](./images/layout_example.png)
+**Responsive Layouts:**
 
-## Anatomy of a Layout Page
+Responsive Layouts are the main layouts in openHAB and recommended for most uses. They automatically adjust to the device's screen and thus can be used as general interface from all devices.
+Within Responsive Layouts you can choose among several types of arrangements, based on whether you want to control the layout completely, or let the container decide where to place the widgets.
 
-A layout page can host one or multiple **blocks**, optionally followed by a **masonry** layout.
+![Responsive Layout Page Example](./images/responsive_layout_example.png)
+
+**Fixed Grid Layouts:**
+
+Fixed Grid Layouts allow you to create a Layout Page where elements have predefined positions and sizes. They are used to design a panel-like page for a specific screen size and are most suitable for static interfaces like wall mounted tablets.
+
+![Fixed Grid Layout Page Example](./images/fixed_grid_layout_example.png)
```

Comment:
If you want to get a working preview of your draft you would have to provide this image or remove the image reference until the image is available and uploaded.